classes/class-wpcom-liveblog-entry.php

```diff
@@ -535,6 +535,38 @@ public static function get_entry_title( $entry ) {
 		return wp_trim_words( $entry->content, 10, '…' );
 	}
 
+	/**
+	 * Get Entry first image if present.
+	 *
+	 * @param  object $entry Entry.
+	 * @return string        Image src.
+	 */
+	public static function get_entry_first_image( $entry ) {
+		$entry_id = ( $entry instanceof WPCOM_Liveblog_Entry ) ? $entry->get_id() : $entry->id;
```

Comment:
If $entry isn't a `WPCOM_Liveblog_Entry`, what is it instead?

Reply:
@paulschreiber ,

`WPCOM_Liveblog::get_entries_paged` returns an array of `stdClass` objects with data as properties.